### DISCUSSION TITLE
$or query operator support

### DISCUSSION THREAD

**Discussion by ryaposov** [marshallswain/feathers-pinia#84] [General]
`$or` query operator does not work for me for `.findInStore` getter. If it supported locally?

I found a test with `associatedFind` using it, however [do not see any related code](https://github.com/marshallswain/feathers-pinia/blob/v1/src/service-store/make-getters.ts#L84) that could make it work.

Could someone clarify? @marshallswain 

Thanks!

**Comment by ryaposov** [comment]
I can see `sift` is used for mongo-like queries, but `$or` never reaches `sift` since it gets filtered out from query 
<img width="625" alt="Screenshot 2022-11-08 at 19 28 19" src="https://user-images.githubusercontent.com/5815992/200647392-30fcdc87-7569-4de0-b80f-0edeb320be53.png">

**Comment by marshallswain** [comment]
@ryaposov thanks for reporting.  This issue has been fixed: https://github.com/marshallswain/feathers-pinia/releases/tag/v1.0.0-pre.23
